legacy_gbdk/gbdk_example/gbt_player_bank1.s

```diff
@@ -1341,7 +1341,7 @@ gbt_ch1234_jump_position:
 	; Check to see if jump puts us past end of song
 	ld	a,(hl)
 	call	gbt_get_pattern_ptr
-	ld	hl,gbt_current_step_data_ptr
+	ld	hl,#gbt_current_step_data_ptr
```

Comment:
Oh, was this change actually needed? I've always hated the syntax of this assembler, to be fair, and I don't get some of the small things like this.

Reply:
According to the rest of the differences between gbt player versions, it seems so.
You have a # to confirm it's a memory address definition, one of the most annoying parts to make a converter.

The :: change is what gets me, weird that :: is full scope to c, but not in scope to asm from another file? Changing that one to : fixed bgb yelling at me about bad op codes, which it never did when calling from asm to :: in the same file.

Reply:
I see. Fair enough. Can you turn this into a real PR that isn't a draft? I could merge it as it is.